package.json

```diff
@@ -6,14 +6,15 @@
     "@testing-library/jest-dom": "^5.16.4",
     "@testing-library/react": "^13.3.0",
     "@testing-library/user-event": "^13.5.0",
+    "big.js": "^6.2.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "react-scripts": "^5.0.1",
+    "react-scripts": "^2.1.3",
     "web-vitals": "^2.1.4"
   },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
+    "start": "react-scripts --openssl-legacy-provider start",
+    "build":"react-scripts --openssl-legacy-provider build",
```

Comment:
- [ ] Kindly remove the `--openssl-legacy-provider` it's preventing me from running the project using npm start.

`hint`
the project run smoothly when I remove it, if it fails in your end kindly check on your node package.